commands/list.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs')
+const tasks = JSON.parse(fs.readFileSync('./tasks.json', 'utf8'))
+
+const list = () => {
+  const longestID = Math.max.apply(null, tasks.map(task => task.id.toString().length))
```

Comment:
This variable name is a little confusing. It's measuring the _number of digits_ in the string for the ID, but could easily be misinterpreted as the "highest ID".

Alternate suggestions `widestIDString` or `maxIDStringLength`